docs/requirements.txt

```diff
@@ -0,0 +1,3 @@
+sphinx==2.3.1
+mkdocs==1.0.4
+numpy==1.18.1
```

Comment:
We could also just fold this into `dev/requirements.txt` and have a single set of Python requirements that cover all development dependencies.

Reply:
I think we can just merge at the moment given that there are not so many dependencies at this moment.